css/style.css

```diff
@@ -0,0 +1,735 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+html,
+body {
+    width: 100%;
+    height: 100%;
+}
+
+html {
+    scroll-padding-top: 80px;
+}
+
+body {
+    min-width: 320px;
+}
+
+.top-nav {
+    width: 100%;
+    position: fixed;
+    top: 0;
+    left: 0;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    background-color: #345162;
+    padding-right: 40px;
+    border-bottom: 2px solid #DAE3E1;
+    z-index: 3;
+}
+
+.nav-links {
+    display: flex;
+    gap: 30px;
+    list-style-type: none;
+}
+
+.nav-links a {
+    text-decoration: none;
+    font-family: "Copse", serif;
+    font-weight: 400;
+    font-style: normal;
+    font-size: 20px;
+    color: #DAE3E1;
+}
+
+.menu-toggle {
+    display: none;
+    background: none;
+    border: none;
+    cursor: pointer;
```

Comment:
Why do you need those properties while the display is none

Reply:
Since the menu icon img resides inside a button element which by default has a border and background, so even if its display is none at this moment, then at some point (CSS breakpoint) its display will be changed to block, so these styles will directly apply. By the way, I will move these styles to apply once the display becomes block, thanks for this note.

Reply:
Resolved.